optuna/trial/_trial.py

```diff
@@ -469,6 +469,10 @@ def objective(trial):
         Raises:
             :exc:`NotImplementedError`:
                 If trial is being used for multi-objective optimization.
+            :exe:`ValueError`:
+                If ``step is not None`` is specified.
```

Comment:
In my understanding, `ValueError` is raised when a negative number is specified to `step`. Could you explain where did you find `None` value regarding ` ValueError`?